### PR TITLE
Adds SPDLog based logging

### PR DESCRIPTION
**Pull request by zeroset**
We need to exchange old logging with new SPDLog based Logger

### CODE REVIEW

File: IO/GeoFileParser.cpp [hunk 1]
```diff
@@ -306,25 +311,25 @@ bool GeoFileParser::LoadGeometry(Building* building)
                p /= transFilename;
                transFilename = p.string();
 
-               Log->Write("INFO:\tParsing transition from file <%s>", transFilename.c_str());
+               Logging::Info(fmt::format("Parsing transition from file <{:s}>", transFilename));
```
Comment (chraibi) [inline IO/GeoFileParser.cpp]: why fmt::format is used here?

Comment (zeroset) [reply IO/GeoFileParser.cpp]: To insert the transFilename into the format string. Before this was done with sprintf

**Comment by zeroset** [comment]
I had to remove some parts from @chraibi because they where wrong and i was unable to merge develop. Please let us merge the SPDLog integration step by step. This can be merged as soon as it is reviewed and tests are done.

File: main.cpp [hunk 2]
```diff
@@ -53,11 +53,11 @@ int main(int argc, char ** argv)
 {
     Logging::Guard guard;
     Logging::Info("Starting JuPedSim - JPScore");
-    Logging::Info(fmt::format("Version {}", JPSCORE_VERSION));
-    Logging::Info(fmt::format("Commit id {}", GIT_COMMIT_HASH));
-    Logging::Info(fmt::format("Commit date {}", GIT_COMMIT_DATE));
-    Logging::Info(fmt::format("Build from branch {}", GIT_BRANCH));
-    Logging::Info(fmt::format("Build with {}({})", compiler_id, compiler_version));
+    Logging::Info(fmt::format(check_fmt("Version {}"), JPSCORE_VERSION));
```
Comment (chraibi) [inline main.cpp]: why do we have to use `check_fmt`?

Comment (zeroset) [reply main.cpp]: Thats the compile time check of the format string. Unfortunately we can not hide this inside a function.

**Pull request merged**